ultimatejobweb/migrations/0002_test_data_company.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations, transaction
+from ultimatejobweb.models import company
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [('ultimatejobweb', '0001_initial')]
+
+    def generate_data(apps, schema_editor):
+
+        companies = [('Facebook', 'https://www.facebook.com/careers/jobs/?q=',
+                      'templates/company logo/FBlogo.png', 'API_FB.py'),
+                     ('Red Hat', 'https://careers-redhat.icims.com/jobs/search?ss=1&in+\
```

Comment:
The `+\` shouldn't be inside the quoted string. And you don't need a `+` either. 
2 strings will automatically combine. See:
```
>>> x = 'abcd' \
... 'efgh'
>>> x
'abcdefgh'

```

Reply:
You actually don't need the newlinw `\` in any case, because it's from within a tuple:
```
>>> tuple_data = ('a', 'bcd'
... 'efg', 'hij')
>>> tuple_data
('a', 'bcdefg', 'hij')

```